File: problems/Warm-Up/Problemset/C/submissions/accepted/solucion.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    int t;
    cin >> t;
    while(t--){
        int n;
        cin >> n;
        vector<int> a(n), b(n);
        for(int i = 0; i < n; i++) cin >> a[i];
        for(int i = 0; i < n; i++) cin >> b[i];

        int iterations = 0;
        while(true){
            bool found1 = false;
            for(int i = 0; i < n; i++){
                if(a[i] > b[i]){
                    a[i]--;
                    found1 = true;
                    break;
                }
            }

            for(int i = 0; i < n; i++){
                if(a[i] < b[i]){
                    a[i]++;
                    break;
                }
            }

            iterations++;

            if(!found1) break;
        }

        cout << iterations << "\n";
    }
    return 0;
}

File: problems/Warm-Up/Problemset/C/submissions/accepted/solucion.py
import sys
input = sys.stdin.readline

t = int(input())
for _ in range(t):
    n = int(input())
    a = list(map(int, input().split()))
    b = list(map(int, input().split()))

    iterations = 0
    while True:
        found1 = False
        for i in range(n):
            if a[i] > b[i]:
                a[i] -= 1
                found1 = True
                break

        for i in range(n):
            if a[i] < b[i]:
                a[i] += 1
                break

        iterations += 1

        if not found1:
            break

    print(iterations)
